SECURITY.md

```diff
@@ -0,0 +1,37 @@
+# Security Policy
+
+## Supported Versions
+
+Security-related reports are considered for official numbered releases
+starting with v3.5. However, issues that do not affect the current Stable or
```

Comment:
As I discussed in 13fc24f, we can change v3.5 to another v3 release if there are good reasons to offer this kind of support for earlier code. I do not suggest any changes here. This change request is just to highlight the issue.

Reply:
We can set that to whatever version we (security team) wants to support, yes.

Today the v3.5 version is the oldest still with snapshots and patch publication active in the automation. So we can easily provide security updates if anyone tests the patch. My practice has typically been to support until that automation turns off.